### PR TITLE
Basic login

### PR DESCRIPTION
**Pull request by tanyakhandelwal21**
This pull request adds login functionality. Known issues:
1. Firebase error handling
2. Input validation
These errors will be fixed in the future.

### CODE REVIEW

File: src/frontend/containers/LoginPage/index.jsx [hunk 1]
```diff
@@ -1,20 +1,60 @@
...
+    };
+
+    // TODO: Handle errors returned by firebase.
+    axios
+      .post('/api/login', credentials)
+      .catch((err) => console.log(err))
+      .then((res) => console.log(res.data));
+
+    // TODO: Make sure the next two lines happen only if login was successful.
+    history.push('/');
```
Comment (drewthoennes) [inline src/frontend/containers/LoginPage/index.jsx:35]: We'll probably want to move this into the `.then` once we build this out further

File: src/frontend/containers/LoginPage/index.jsx [hunk 2]
```diff
@@ -1,20 +1,60 @@
...
+    localStorage.setItem('currentUser', document.getElementById('email').value);
```
Comment (drewthoennes) [inline src/frontend/containers/LoginPage/index.jsx:36]: What all are we using this value for? If it's for storing the state, we do have Redux

File: src/frontend/containers/RegisterPage/index.jsx [hunk 3]
```diff
@@ -18,22 +17,28 @@ class RegisterPage extends React.Component {
   }
 
   handleRegister() {
+    /* This method handles registration of a new user by sending the user credentials to the
+        corresponding endpoint and redirecting to the login page. */
+    // TODO: Validate credentials.
     const credentials = {
       email: document.getElementById('email').value,
-      password: passwordHash.generate(document.getElementById('password').value),
+      password: btoa(document.getElementById('password').value),
```
Comment (drewthoennes) [inline src/frontend/containers/RegisterPage/index.jsx:25]: Did you intend to use a function that doesn't hash and salt the password? It seems `btoa` just encodes this value for transfer

**Review by drewthoennes — CHANGES_REQUESTED** [review]

File: src/frontend/containers/RegisterPage/index.jsx (hunk 3 above)
Comment (tanyakhandelwal21) [reply src/frontend/containers/RegisterPage/index.jsx:25]: Yes, firebase hashes the passwords so I just wanted to encode it so I don't send the password to the backend in plain text.

File: src/frontend/containers/LoginPage/index.jsx (hunk 2 above)
Comment (tanyakhandelwal21) [reply src/frontend/containers/LoginPage/index.jsx:36]: Oh right, we don't need to store it then!

File: src/frontend/containers/RegisterPage/index.jsx (hunk 3 above)
Comment (drewthoennes) [reply src/frontend/containers/RegisterPage/index.jsx:25]: Sounds good! Encoding is different than encrypting, so with or without `btoa`, we'd be sending a plain text password to our backend. As long as we're using SSL (which Heroku is able to), we don't have to worry about this

**Review by drewthoennes — APPROVED** [review]

**Pull request merged**